<details>
<summary>build log (aarch64-darwin)</summary>

```
❯ nixpkgs-review pr --post-result 345299
$ git -c fetch.prune=false fetch --no-tags --force https://github.com/NixOS/nixpkgs master:refs/nixpkgs-review/0 pull/345299/head:refs/nixpkgs-review/1
$ git worktree add /Users/momeemt/.cache/nixpkgs-review/pr-345299-1/nixpkgs 6710d0dd013f55809648dfb1265b8f85447d30a6
Preparing worktree (detached HEAD 6710d0dd013f)
Updating files: 100% (43452/43452), done.
HEAD is now at 6710d0dd013f nixos/logrotate: relax hardening (#345275)
$ nix-env --extra-experimental-features no-url-literals --option system aarch64-darwin -f <nixpkgs> --nix-path nixpkgs=/Users/momeemt/.cache/nixpkgs-review/pr-345299-1/nixpkgs nixpkgs-overlays=/var/folders/09/9ttjqns51hx4cl_3c96llt_r0000gp/T/tmp57e37w2f -qaP --xml --out-path --show-trace --no-a
llow-import-from-derivation
$ git merge --no-commit --no-ff 1fc29db8ec6bcb78057fa9eb5d42ad675eb14d17
Automatic merge went well; stopped before committing as requested
$ nix-env --extra-experimental-features no-url-literals --option system aarch64-darwin -f <nixpkgs> --nix-path nixpkgs=/Users/momeemt/.cache/nixpkgs-review/pr-345299-1/nixpkgs nixpkgs-overlays=/var/folders/09/9ttjqns51hx4cl_3c96llt_r0000gp/T/tmp57e37w2f -qaP --xml --out-path --show-trace --no-a
llow-import-from-derivation --meta
2 packages updated:
rakshasa-rtorrent xmlrpc_c (1.51.07 → 1.59.03)

$ nix build --file /nix/store/dz06k69xdrhw0cgm7a9irmg27sva0v79-nixpkgs-review-2.10.5/lib/python3.12/site-packages/nixpkgs_review/nix/review-shell.nix --nix-path 'nixpkgs=/Users/momeemt/.cache/nixpkgs-review/pr-345299-1/nixpkgs nixpkgs-overlays=/var/folders/09/9ttjqns51hx4cl_3c96llt_r0000gp/T/tm
p57e37w2f' --extra-experimental-features 'nix-command no-url-literals' --no-link --keep-going --no-allow-import-from-derivation --argstr system aarch64-darwin --argstr nixpkgs-path /Users/momeemt/.cache/nixpkgs-review/pr-345299-1/nixpkgs --argstr nixpkgs-config-path /var/folders/09/9ttjqns51hx4
cl_3c96llt_r0000gp/T/tmpxmp8t5at.nix --argstr attrs-path /Users/momeemt/.cache/nixpkgs-review/pr-345299-1/attrs.nix
error: builder for '/nix/store/x4s9sryd734rg8gnmhk33368lbgq8xkq-xmlrpc-c-1.59.03.drv' failed with exit code 2;
       last 10 log lines:
       > clang -c -o curltransaction.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3 -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.0
3 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include   curltransaction.c
       > clang -c -o curlmulti.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3 -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/p
rivate/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include   curlmulti.c
       > clang -c -o curltransaction.osh -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include -DND
EBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3  -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.0
3.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include    curltransaction.c
       > clang -c -o curlmulti.osh -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include -DNDEBUG -
pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3  -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-
0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include    curlmulti.c
       > clang -c -o xmlrpc_curl_transport.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3 -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-
1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include   xmlrpc_curl_transport.c
       > clang -c -o xmlrpc_curl_transport.osh -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/includ
e -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3  -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-
1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include    xmlrpc_curl_transport.c
       > make[2]: Leaving directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/curl_transport'
       > make[2]: Leaving directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/util'
       > make[1]: Leaving directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib'
       > make: *** [/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/common.mk:381: lib/all] Error 2
       For full logs, run 'nix-store -l /nix/store/x4s9sryd734rg8gnmhk33368lbgq8xkq-xmlrpc-c-1.59.03.drv'.
error: 1 dependencies of derivation '/nix/store/7m1p63s6xga617g7pabcqq7wmz8ps454-rakshasa-rtorrent-0.9.8-unstable-2024-09-07.drv' failed to build
error: 2 dependencies of derivation '/nix/store/3bdc9b56b3gjg9y8gvq4b4ybmwskywkz-review-shell.drv' failed to build

Link to currently reviewing PR:
https://github.com/NixOS/nixpkgs/pull/345299

3 packages failed to build:
rtorrent rtorrent.man xmlrpc_c

error: build log of '/nix/store/7m1p63s6xga617g7pabcqq7wmz8ps454-rakshasa-rtorrent-0.9.8-unstable-2024-09-07.drv^*' is not available
error: build log of '/nix/store/v00qyz1w60z0xkx673lj0c7dafvzrf0z-rakshasa-rtorrent-0.9.8-unstable-2024-09-07' is not available
error: build log of '/nix/store/7m1p63s6xga617g7pabcqq7wmz8ps454-rakshasa-rtorrent-0.9.8-unstable-2024-09-07.drv^*' is not available
error: build log of '/nix/store/askivf0rmk0xf7maql60fhn7i51xd8fc-rakshasa-rtorrent-0.9.8-unstable-2024-09-07-man' is not available
Posting result comment on https://github.com/NixOS/nixpkgs/pull/345299
```
</details>

<details>
<summary>`nix-store -l /nix/store/x4s9sryd734rg8gnmhk33368lbgq8xkq-xmlrpc-c-1.59.03.drv 2&>1 | pbcopy`</summary>
```
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/lcp9j1nfs631kr60g06rbc7lpw8mrk2s-xmlrpc-c-1.59.03.tgz
source root is xmlrpc-c-1.59.03
setting SOURCE_DATE_EPOCH to timestamp 1711386275 of file xmlrpc-c-1.59.03/src/xmlrpc_struct.c
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Updating Autotools / GNU config script to a newer upstream version: ./config.sub
Updating Autotools / GNU config script to a newer upstream version: ./config.guess
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
patching script interpreter paths in ./configure
./configure: interpreter directive changed from "#! /bin/sh" to "/nix/store/iafzjk7zbkqaszqfp6n006vvxjrpn4f6-bash-5.2p32/bin/sh"
configure flags: --prefix=/nix/store/4bi3k3vpssym647krdqwiicg9lrnlhwc-xmlrpc-c-1.59.03 --enable-libxml2-backend
checking for a BSD-compatible install... /nix/store/1l41471x6rlf7l544s1bxkqqgpprd28m-coreutils-9.5/bin/install -c
checking whether build environment is sane... yes
checking whether make sets $(MAKE)... yes
checking for working aclocal... missing
checking for working autoconf... missing
checking for working automake... missing
checking for working autoheader... missing
checking for working makeinfo... missing
checking build system type... aarch64-apple-darwin23.5.0
checking host system type... aarch64-apple-darwin23.5.0
checking for wininet-config... no
configure: You don't appear to have Wininet installed (no working wininet-config in your command search path), so we will not build the Wininet client XML transport
checking whether to build Wininet client XML transport module... no
checking for curl-config... yes
checking whether to build Curl client XML transport module... yes
checking for libwww-config... no
configure: You don't appear to have Libwww installed (no working libwww-config in your command search path), so we will not build the Libwww client XML transport
checking whether to build Libwww client XML transport module... no
checking for gcc... clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether clang accepts -g... yes
checking for clang option to accept ISO C89... none needed
checking for main in -lncurses... no
checking for main in -lreadline... no
checking whether to build tools... yes
checking whether to build the xmlrpc_pstream tool... no
checking whether to build Abyss server module... yes
checking whether to build CGI server module... yes
checking whether to build C++ wrappers and tools... yes
checking for gcc... (cached) clang
checking whether we are using the GNU C compiler... (cached) yes
checking whether clang accepts -g... (cached) yes
checking for clang option to accept ISO C89... (cached) none needed
checking whether we are using the GNU C++ compiler... yes
checking whether clang++ accepts -g... yes
checking for socket... yes
checking how to run the C preprocessor... clang -E
checking for grep that handles long lines and -e... /nix/store/p57w57nln7yjvj95i9wlx27g7gcnwgga-gnugrep-3.11/bin/grep
checking for egrep... /nix/store/p57w57nln7yjvj95i9wlx27g7gcnwgga-gnugrep-3.11/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking wchar.h usability... yes
checking wchar.h presence... yes
checking for wchar.h... yes
checking sys/filio.h usability... yes
checking sys/filio.h presence... yes
checking for sys/filio.h... yes
checking sys/ioctl.h usability... yes
checking sys/ioctl.h presence... yes
checking for sys/ioctl.h... yes
checking sys/select.h usability... yes
checking sys/select.h presence... yes
checking for sys/select.h... yes
checking stdarg.h usability... yes
checking stdarg.h presence... yes
checking for stdarg.h... yes
checking for size_t... yes
checking whether va_list is an array... no
checking whether compiler has __attribute__... yes
checking for vsnprintf... yes
checking for wcsncmp... yes
checking for setgroups... yes
checking for asprintf... yes
checking for setenv... yes
checking for strtoll... yes
checking for strtoull... yes
checking for strtoq... yes
checking for strtouq... yes
checking for __strtoll... no
checking for __strtoull... no
checking for _strtoui64... no
checking for pselect... yes
checking for gettimeofday... yes
checking for localtime_r... yes
checking for gmtime_r... yes
checking for strcasecmp... yes
checking for stricmp... no
checking for _stricmp... no
checking whether to use Abyss pthread function... yes
checking for curl-xmlrpc-config... no
checking for curl-config... /nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/bin/curl-config
checking for Curl library directory... /nix/store/dijbmwxxhizpgva2idb261ic4f0r47cq-curl-8.9.1/lib
checking for OpenSSL library... ./configure: line 5991: pkg-config: command not found
no
configure: You don't appear to have Openssl installed (no pkg-config file for it in your pkg-config search path), so we will not build the Abyss Openssl channel module
checking whether to build Abyss Openssl channel module... no
checking whether to use SSL with libwww... no
checking whether to build the libxml2 backend... yes
checking for xml2-config... yes
checking for ar... ar
checking for ranlib... ranlib
configure: creating ./config.status
config.status: creating srcdir.mk
config.status: creating config.mk
config.status: creating xmlrpc_config.h
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j10 SHELL=/nix/store/iafzjk7zbkqaszqfp6n006vvxjrpn4f6-bash-5.2p32/bin/bash
rm -f shell_config
make -C include/ -f /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03//include/Makefile \
    all 
Lots of echoes to 'shell_config' suppressed here ...
make[1]: Entering directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include'
>xmlrpc-c/config.h
Lots of echoes to 'xmlrpc-c/config.h' suppressed here ...
rm -f xmlrpc-c-config
rm -f xmlrpc-c-config.test
Echoes to 'xmlrpc-c-config.test' suppressed here ...
Echoes to 'xmlrpc-c-config' suppressed here ...
cat shell_config >>xmlrpc-c-config
cat shell_config >>xmlrpc-c-config.test
cat xmlrpc-c-config.test.main >>xmlrpc-c-config.test
cat xmlrpc-c-config.main >>xmlrpc-c-config
chmod a+rx xmlrpc-c-config
chmod a+rx xmlrpc-c-config.test
make[1]: Leaving directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include'
make -C lib/ -f /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03//lib/Makefile \
    all 
make[1]: Entering directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib'
make -C util/ -f /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/util/Makefile \
    all 
make -C libutil/ -f /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/libutil/Makefile \
    all 
make -C /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/ -f /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/GNUmakefile version.h
make[2]: Entering directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/util'
ln -s /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 srcdir
ln -s /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 blddir
make[2]: Entering directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/libutil'
ln -s /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 srcdir
ln -s /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 blddir
make[2]: Entering directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03'
rm -f version.h
cat /dev/null >depend.mk
cat /dev/null >depend.mk
echo "/* Generated by make file rule */" >>version.h
echo "#define XMLRPC_C_VERSION" \"1.59.2"\"" >>version.h
echo "#define XMLRPC_VERSION_MAJOR 1" >>version.h
clang -c -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -Isrcdir/lib/util/include -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03   casprintf.c
clang -c -o asprintf.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -Isrcdir -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include   asprintf.c
clang -c -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -Isrcdir/lib/util/include -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03   cmdline_parser.c
clang -c -o base64.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -Isrcdir -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include   base64.c
clang -c -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -Isrcdir/lib/util/include -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03   getoptx.c
clang -c -o error.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -Isrcdir -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include   error.c
clang -c -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -Isrcdir/lib/util/include -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03   string_parser.c
clang -c -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -Isrcdir/lib/util/include -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03   stripcaseeq.c
clang -c -o lock_platform.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -Isrcdir -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include   lock_platform.c
echo "#define XMLRPC_VERSION_MINOR 59" >>version.h
echo "#define XMLRPC_VERSION_POINT 2" >>version.h
make[2]: Leaving directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03'
clang++ -c -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas  -Wsynth -g   -Isrcdir/lib/util/include -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03   cmdline_parser_cpp.cpp
clang -c -o lock_pthread.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -Isrcdir -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include   lock_pthread.c
[1mcasprintf.c:59:1: [0m[0;1;35mwarning: [0m[1munused function 'simpleVasprintf' [-Wunused-function][0m
simpleVasprintf(char **      const resultP,
[0;1;32m^
[0m[1masprintf.c:116:19: [0m[0;1;35mwarning: [0m[1ma function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes][0m
xmlrpc_strnomemval() {
[0;1;32m                  ^
[0m[0;32m                   void
[0m[1masprintf.c:132:10: [0m[0;1;31merror: [0m[1mcall to undeclared function 'vasprintf'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration][0m
    rc = vasprintf(&string, fmt, varargs);
[0;1;32m         ^
[0m[1masprintf.c:132:10: [0m[0;1;30mnote: [0mdid you mean 'vsprintf'?[0m
[1m/nix/store/apgxk5abmyyc4b8xydd5ghf4fpb5shkl-libSystem-11.0.0/include/stdio.h:192:6: [0m[0;1;30mnote: [0m'vsprintf' declared here[0m
int      vsprintf(char * __restrict, const char * __restrict, va_list) __printflike(2, 0) __swift_unavailable("Use vsnprintf instead.");
[0;1;32m         ^
[0m1 warning and 1 error generated.
clang -c -o lock_none.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3   -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -Isrcdir -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include   lock_none.c
1 warning generated.
make[2]: *** [/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/common.mk:342: asprintf.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make -C curl_transport/ -f /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/curl_transport/Makefile \
    all 
make[2]: Entering directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/curl_transport'
ln -s /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 srcdir
ln -s /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 blddir
cat /dev/null >depend.mk
make[2]: Leaving directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/libutil'
make[1]: *** [/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/common.mk:381: libutil/all] Error 2
make[1]: *** Waiting for unfinished jobs....
ln -s /private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/version.h version.h
clang -c -o curltransaction.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3 -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include   curltransaction.c
clang -c -o curlmulti.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3 -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include   curlmulti.c
clang -c -o curltransaction.osh -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3  -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include    curltransaction.c
clang -c -o curlmulti.osh -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3  -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include    curlmulti.c
clang -c -o xmlrpc_curl_transport.o -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3 -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include   xmlrpc_curl_transport.c
clang -c -o xmlrpc_curl_transport.osh -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include -DNDEBUG -pthread -Wall -W -Wno-uninitialized -Wundef -Wno-unknown-pragmas -Wmissing-declarations -Wstrict-prototypes -Wmissing-prototypes -Wimplicit -fno-common -g -O3  -DHAVE_CURL_MULTI_POLL  -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03 -I/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/include -Isrcdir/include -Isrcdir/lib/util/include -I/nix/store/g7iznpcciawm2alm7hn9qf2x2nz9gcld-curl-8.9.1-dev/include    xmlrpc_curl_transport.c
make[2]: Leaving directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/curl_transport'
make[2]: Leaving directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib/util'
make[1]: Leaving directory '/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/lib'
make: *** [/private/tmp/nix-build-xmlrpc-c-1.59.03.drv-0/xmlrpc-c-1.59.03/common.mk:381: lib/all] Error 2
```
</details>

I think this following section is the most important of the build log.

```
asprintf.c:132:10: error: call to undeclared function 'vasprintf'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
    rc = vasprintf(&string, fmt, varargs);
         ^
asprintf.c:132:10: note: did you mean 'vsprintf'?
/nix/store/apgxk5abmyyc4b8xydd5ghf4fpb5shkl-libSystem-11.0.0/include/stdio.h:192:6: note: 'vsprintf' declared here
int      vsprintf(char * __restrict, const char * __restrict, va_list) __printflike(2, 0) __swift_unavailable("Use vsnprintf instead.");
```

The version of c compiler or toolchains appears to be the cause?

--- a/pkgs/development/libraries/xmlrpc-c/default.nix
+++ b/pkgs/development/libraries/xmlrpc-c/default.nix
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "xmlrpc-c";
-  version = "1.51.07";
+  version = "1.59.03";
 
   src = fetchurl {
     url = "mirror://sourceforge/xmlrpc-c/${pname}-${version}.tgz";
-    sha256 = "sha256-hNIK4z+SdYL4IdYcC5GUrvvx15JFkKE/qdpa4WmK3tk=";
+    hash = "sha256-vbcdtCqwvlFZFVWIXRFoKwRMEDTUoylkAb+SHsCyM/4=";
   };
 
   buildInputs = [ curl libxml2 ];
@@ -20,7 +20,10 @@ stdenv.mkDerivation rec {
     (cd tools/xmlrpc && make && make install)
   '';
 
-  hardeningDisable = [ "format" ];
+  enableParallelBuilding = true;
+
+  # ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang "-Wno-error=implicit-function-declaration";
 
   meta = with lib; {
     description = "Lightweight RPC library based on XML and HTTP";

